hugegraph-pd/hg-pd-dist/src/assembly/static/conf/log4j2.xml

```diff
@@ -129,7 +129,7 @@
         <!-- Use mixed async way to output logs -->
         <logger name="org.apache.hugegraph" level="INFO" additivity="false">
             <appender-ref ref="file" />
-            <appender-ref ref="console" />
+<!--            <appender-ref ref="console" />-->
```

Comment:
why we need disable the output to console?

Reply:
I'm just considering redundancy. My understanding is that only the console appender was used in the org.apache.huggegraph logger, and it was duplicated with the file appender, so only one was kept? So I kept the file appender

Reply:
It needs to be discussed in different situations because this configuration file will be used by both the IDE launch and script usage:

- If started using a script, take pd as an example, the output of the command line will be redirected to hugegraph-pd-stdout.log, and the logs defined within the log4j2.xml range will be obtained in hugegraph-pd.log (there will be a delay). In fact, the content of the `hugegraph-pd.log` log is basically the same as that in `hugegraph-pd-stdout.log`.
- If started using an IDE, still using pd as an example, the command line output (pd runtime log) will be displayed in the RUN panel, and you will still get the logs within the range defined in log4j2.xml in the hugegraph-pd.log (there will be a delay).

The conclusion is that if the script is used to start, the console appender is unnecessary. If the IDE is used to start, the console appender is meaningful (for timely observation of logs). Considering the delay in the generation of hugegraph-pd.log logs, it is recommended to keep the console appender.

Reply:
Refer to the startup configuration of IDE for PD as follows:

```text
-Dname=HugeGraphPD
-Xms512m
-Xmx6889m
-XX:+HeapDumpOnOutOfMemoryError
-XX:HeapDumpPath=/home/vgalaxies/Desktop/incubator-hugegraph/hugegraph-pd/apache-hugegraph-incubating-pd-1.5.0/logs
-Dspring.config.location=/home/vgalaxies/Desktop/incubator-hugegraph/hugegraph-pd/apache-hugegraph-incubating-pd-1.5.0/conf/application.yml
-Dlog4j.configurationFile=/home/vgalaxies/Desktop/incubator-hugegraph/hugegraph-pd/apache-hugegraph-incubating-pd-1.5.0/conf/log4j2.xml
```